### PR TITLE
Add information for macOS into the user docs

### PR DESCRIPTION
**Pull request by SueChaplain**
Modify command line topics to indicate which ones
do/don't apply to OS X, plus any specific information
for this platform.

Updates also for the directory conventions topic,
environment variables topic, and messages.

Closes: #82

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
@babsingh - I think I have got everything in based on your updated spreadsheet, so this should be good for 0.12.0. I know there are a few questions outstanding that you've raised issues for, so we can handle those as and when that work completes.

Please can you take a look through the files. If you want to see the built doc for Eclipse, plse ping me and I can share a private draft.

**Comment by babsingh** [comment]
Yes, I will go through the files. It will be easier to review the files with the built doc for Eclipse. Requesting @pshipton or @DanHeidinga to be the second set of eyes.

@SueChaplain Created a group DM session on Slack. Requested a private draft of the built doc over there.

File: docs/cmdline_specifying.md [hunk 1]
```diff
@@ -81,31 +80,26 @@ At startup, the list of VM arguments is constructed in the following order, with
...
 
 4.  The `OPENJ9_JAVA_OPTIONS` environment variable. You can set command-line options using this environment variable. The options that you specify with this environment variable are added to the command line when a VM starts in that environment. The environment variable can contain multiple blank-delimited argument strings, but must not contain comments. For example:
 
     - On Windows systems:
 
-            :::java
             set OPENJ9_JAVA_OPTIONS="-Dmysysprop1=tcpip -Dmysysprop2=wait -Xdisablejavadump"
 
-    - On AIX, Linux, and z/OS systems:
+    - On AIX, Linux, macOS, and z/OS systems:
```
[the remark this answers is not in the document]
Comment (SueChaplain) [reply docs/cmdline_specifying.md]: It is standard practice to trademark the first appearance of a term on a page, but not subsequent ones.

File: docs/env_var.md [hunk 2]
```diff
@@ -46,8 +46,8 @@ Use values exactly as shown in the documentation. The names of environment varia
 To set the environment variable **LOGIN\_NAME** to *Fred*, run:
 
 -   `set LOGIN_NAME=Fred` (Windows)
--   `export LOGIN_NAME=Fred` (AIX/Linux: ksh or bash shells)
--   `setenv LOGIN_NAME Fred` (macOS or csh shells)
+-   `export LOGIN_NAME=Fred` (AIX/Linux/macOS: ksh or bash shells)
```
[the remark this answers is not in the document]
Comment (SueChaplain) [reply docs/env_var.md]: First instances are trademarked. I think this file is fine.

File: docs/openj9_support.md [hunk 3]
```diff
@@ -102,9 +102,9 @@ minimum glibc version 2.12 are expected to function without problems.
 | Windows Server 2012 R2     |   Y    |   Y   |
 | Windows Server 2016        |   Y    |   Y   |
 
-| macOS&reg;             |   x64    |
-|------------------------|----------|
-| OSX 10.10+             |    Y     |
+| macOS&reg;              |   x64    |
+|-------------------------|----------|
+| OS X&reg; 10.10+        |    Y     |
```
[the remark this answers is not in the document]
Comment (SueChaplain) [reply docs/openj9_support.md]: Updated

File: docs/openj9_support.md [hunk 4]
```diff
@@ -179,9 +179,9 @@ minimum glibc version 2.12 are expected to function without problems.
 | Windows Server 2012 R2     |   Y    |
 | Windows Server 2016        |   Y    |
 
-| macOS                  |   x64    |
-|------------------------|----------|
-| OSX 10.10+             |    Y     |
+| macOS                   |   x64    |
+|-------------------------|----------|
+| OS X 10.10+             |    Y     |
```
[the remark this answers is not in the document]
Comment (SueChaplain) [reply docs/openj9_support.md]: Updated

File: docs/xgcpolicy.md [hunk 5]
```diff
@@ -123,7 +123,7 @@ The following options are ignored when specified with `-Xgcpolicy:balanced`:
 
 : The generational concurrent policy (default) uses a concurrent mark phase combined with generational garbage collection to help minimize the time that is spent in any garbage collection pause. This policy is particularly useful for applications with many short-lived objects, such as transactional applications. Pause times can be significantly shorter than with the `optthruput` policy, while still producing good throughput. Heap fragmentation is also reduced.
 
-### `metronome` (AIX&reg;, Linux&reg; only)
+### `metronome` (AIX&reg;, Linux&reg; x86 only)
```
[the remark this answers is not in the document]
Comment (SueChaplain) [reply docs/xgcpolicy.md]: Yes! Fixed, thank you.

File: docs/xlpobjectheap.md [hunk 6]
```diff
@@ -59,6 +59,8 @@ See [Using -X command-line options](x_jvm_commands.md) for more information abou
     - Linux on IBM Z&reg;: 1 MB large pages
     - On other architectures, the VM uses the default operating system page size.
 
+: On macOS, the default page size is 4 K.
```
[the remark this answers is not in the document]
Comment (SueChaplain) [reply docs/xlpobjectheap.md]: Agreed.

File: docs/xgc.md [hunk 7]
```diff
@@ -152,8 +152,7 @@ Options that change the behavior of the Garbage Collector (GC).
 
 : where, `<address>` is the base memory address for the heap. Use this option with the `-Xcompressedrefs` option to allocate the heap you specify with the [`-Xmx`](xms.md) option, in a memory range of your choice. If `-Xcompressedrefs` is not specified, this option has no effect. In the following example, the heap is located at the 4 GB mark, leaving the lowest 4 GB of address space for use by other processes.
 
-        :::java
-        -Xgc:preferredHeapBase=0x100000000
+      -Xgc:preferredHeapBase=0x100000000
```
[the remark this answers is not in the document]
Comment (SueChaplain) [reply docs/xgc.md]: It lost a tab, fixed.

**Pull request merged**